var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -100,3 +109,7 @@ def configure_args(self):
         ])
 
         return options
+
+    def build(self, spec, prefix):
+        with working_dir(self.build_directory):
+            make(*self.build_targets, parallel=False)
```

Comment:
Why is this method overridden? The only difference I see from:

https://github.com/spack/spack/blob/1398038beead1a1532fc337165e2b6bf9d58c284/lib/spack/spack/build_systems/autotools.py#L324-L329

is `parallel=False`. Wouldn't an attribute at class level achieve the same effect?

Reply:
Thanks, wasn't aware that was a thing